docs/docs/limits/README.md

```diff
@@ -89,10 +89,6 @@ Currently, most of the [limits that apply to HTTP triggers](#http-triggers) also
 
 The only limit that differs between email and HTTP triggers is the payload size: the body of HTTP requests is limited to `{{$site.themeConfig.PAYLOAD_SIZE_LIMIT}}`, where the total size of an email sent to a workflow - its body, headers, and attachments - is limited to `{{$site.themeConfig.EMAIL_PAYLOAD_SIZE_LIMIT}}` on the default interface.
```

Comment:
You can probably remove "on the default interface" at the end of the last sentence, since there's only a single interface now.

Reply:
@ctrlaltdylan Did you see the comments in this file?